examples/dts/README.md

```diff
@@ -0,0 +1,55 @@
+# Examples
+
+This directory contains examples of how to author durable orchestrations using the Durable Task Python SDK in conjunction with the Durable Task Scheduler (DTS).
+
+## Prerequisites
+
+All the examples assume that you have a Durable Task Scheduler taskhub created.
+
+The simplest way to create a taskhub is by using the az cli commands:
+
+1. Create a scheduler:
+    az durabletask scheduler create --resource-group <testrg> --name <testscheduler> --location <eastus> --ip-allowlist "[0.0.0.0/0]" --sku-capacity 1 --sku-name "Dedicated" --tags "{}"
+
+1. Create your taskhub
+
+    ```bash
+    az durabletask taskhub create --resource-group <testrg> --scheduler-name <testscheduler> --name <testtaskhub>
+    ```
+
+1. Retrieve the endpoint for the scheduler. This can be done by locating the taskhub in the portal.
+
+1. Set the appropriate environment variables for the TASKHUB and ENDPOINT
+
+    ```bash
+    export TASKHUB=<taskhubname>
+    export ENDPOINT=<taskhubEndpoint>
+    ```
+
+1. Since the samples rely on azure identity, ensure the package is installed and up-to-date
+
+    ```bash
+    python3 -m pip install azure-identity
+    ```
+
+1. Install the correct packages from the top level of this repository, i.e. durabletask-python/

```

Comment:
These are development instructions -- but wouldn't it be better to tell people to install from PyPI using `pip install durabletask` and `pip install durabletask-azuremanaged` ?

The instructions here would install and unreleased version from main branch -- we definitely will not provide support for installing from `main` branch. Only artifacts released to PyPI.

So I think making this clear would be worthwhile @RyanLettieri

Reply:
I'm not sure how up-to-date we keep these packages. I'll specify that this is installing from main as that is what the top level README.md does

Reply:
Customers should not be installing from main. Only contributors should be doing so - or folks who want to live on the edge and don't need support.

Also keep in mind that customers would never see any changes unless they checked out changes from this repo first. Pulling updates from PyPI is easier.

We will definitely want to publish this package frequently enough to PyPI - but keep in mind that we need to also support the packages we publish - which is easier to do than having to provide official support for every single commit in this repo.

Thoughts @torosent @cgillum ?

Reply:
Yep, PyPI will be the only officially supported way to use our libraries.